### ISSUE TITLE
Client side render failed with react_ujs error

### ISSUE THREAD

**Issue by zetachang** [ruby-hyperloop/hyper-react#172]
Got this while using with rails integration. 

```
react_ujs_mount.self-5293119….js?body=1:84 Uncaught Error: Cannot find component: 'React.TopLevelRailsComponent'. Make sure your component is globally available to render.
```

**Comment by catmando** [comment]
did you get it fixed?  I have never seen that, but it I think it means what it says... seems like a config issue.

**Comment by zetachang** [comment]
No luck, I think I've screwed up with my setup with rails. Will get back to this later.

**Comment by rewnad** [comment]
@zetachang I just ran into this issue as well, did you end up getting it fixed?

**Comment by catmando** [comment]
@schoolandrew any chance u could push your project to a repo.  I cannot reproduce, so it would be good to have example

**Comment by sfcgeorge** [comment]
I just got this trying to render a component using the [helper](https://github.com/ruby-hyperloop/hyper-react/blob/master/DOCS.md#elements-and-rendering) - not via the App component.

```erb
<div>
  <%= react_component("Foo") %>
</div>
```

I got other errors like "uninitialized constant React::Component::Callbacks" when rendering via routing.

For me this was caused by attempting to use Opal 0.11 so I have opened that incompatibility as [another issue](https://github.com/ruby-hyperloop/hyper-react/issues/222).
